### DISCUSSION TITLE
Welcome to ClashT Discussions!

### DISCUSSION THREAD

**Discussion by LOVAHE** [DryPeng/clashT#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

However, to ensure everyone’s rights and safety, we hope that everyone can comply with:
 * Treat everyone with respect and have patience.
 * Do not post age-restricted or obscene content.
 * Do not share personal information.
 * Follow [GitHub Terms](https://docs.github.com/en/site-policy/github-terms).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by flawrist** [comment]
Good afternoon, dear developers and community members.
Could you be so kind as to tell where can I find ClashT docs?
Both links mentioned in [README.md](https://github.com/DryPeng/clashT?tab=readme-ov-file#documentation) are dead, error 404.

Update: I found some docs at https://en.clash.wiki/.

**Reply by LOVAHE** [reply]
Hello!

You can access [here](https://docs.drypeng.io/clash/).

**Reply by flawrist** [reply]
Re: @LOVAHE 

Your link is dead, 404.
